README.md

```diff
@@ -1674,6 +1676,40 @@ Manages an SNMP notification on a Cisco SNMP server.
 Determine whether the trap should be on or off. Valid
 values are true and false.
 
+### Type: snmp_notification_receiver
+
+Manages an SNMP user on an cisco SNMP server.
+
+#### Parameters
+
+##### `ensure`
+Determines whether the config should be present or not on the device. Valid
+values are 'present', and 'absent'.
+
+##### `name`
+Name of the SNMP user. Valid value is a string.
```

Comment:
The type indicates that this is `Hostname or IP address of the receiver`  Which description should be used here?